### PR TITLE
Fix tooltip capitalization

### LINKED ISSUE #7515: Tooltip capitalization inconsistencies across the game
**Describe the bug:** 

There are many inconsistencies in capitalization of tooltip texts across the game which should be unified.

**Screenshots or videos showing encountered issue:** 

Here are a few that I've found:

Mixed case:

image

image

Title case:

image

Lowercase:

image

image

**osu!lazer version:** 2020.112.0

### PR DESCRIPTION
**Pull request by revelsix**
closes #7515

### CODE REVIEW

**Comment by revelsix** [comment]
bruh i forgot mods, 1 sec

File: osu.Game/Overlays/Chat/ChatLine.cs [hunk 1]
```diff
@@ -236,7 +236,7 @@ public MenuItem[] ContextMenuItems
                 {
                     List<MenuItem> items = new List<MenuItem>
                     {
-                        new OsuMenuItem("View Profile", MenuItemType.Highlighted, Action)
+                        new OsuMenuItem("view profile", MenuItemType.Highlighted, Action)
```
Comment (iiSaLMaN) [inline osu.Game/Overlays/Chat/ChatLine.cs]: menu items are not tooltips, although I wouldn't mind making them lowercase (osu! text style is mostly in lowercase), but in a different PR please.

File: osu.Game/Users/UserPanel.cs [hunk 2]
```diff
@@ -247,7 +247,7 @@ private void displayStatus(UserStatus status, UserActivity activity = null)
 
         public MenuItem[] ContextMenuItems => new MenuItem[]
         {
-            new OsuMenuItem("View Profile", MenuItemType.Highlighted, ViewProfile),
+            new OsuMenuItem("view profile", MenuItemType.Highlighted, ViewProfile),
```
Comment (iiSaLMaN) [inline osu.Game/Users/UserPanel.cs]: same as above.

**Review by frenzibyte — CHANGES_REQUESTED** [review]

**Review by jcspeet — APPROVED** [review]

**Comment by peppy** [comment]
Blocked until further discussion happens regarding consistency with `osu-web` (which as mentioned, uses title case for tooltips).

**Review by peppy — APPROVED** [review]
Looks like we're going with lower case for now.

**Pull request merged**